plugins_/syntaxtest_dev.py

```diff
@@ -42,7 +42,7 @@ def get_syntax_test_tokens(view):
     match = None
     if line.size() < 1000:  # no point checking longer lines as they are unlikely to match
         first_line = view.substr(line)
-        match = re.match(r'^(?P<comment_start>\s*\S+)'
+        match = re.match(r'^(?P<comment_start>\s*.+)'
                          r'\s+SYNTAX TEST\s+'
```

Comment:
This regex will match extraneous whitespace (any more than one) between the syntax test marker and the `SYNTAX TEST` token as part of the marker. Does this represent ST's internal behavior or should the any match be made non-greedy? (i.e. `\s*.+?`)

Reply:
Good question, I will investigate that a bit more and get back to you, thanks!

Reply:
you were correct - it should be non-greedy. I have pushed a new commit to fix this.